lib/broadway.ex

```diff
@@ -500,7 +502,7 @@ defmodule Broadway do
         ]
       ],
       batchers: [
-        required: true,
+        required: false,
```

Comment:
Can we say `default: []` here so we don't need to do `batchers_config || []` later on?